application/microservices/entando-hub-catalog-ms/src/test/java/com/entando/hub/catalog/integration/BundleGroupFlowIT.java

```diff
@@ -235,12 +235,14 @@ void shouldUpdateBundleGroup() throws Exception {
     @Test
     @WithMockUser(roles={ADMIN})
     void shouldReturnNotFoundWhenUpdateBundleGroupThatDoesntExist() throws Exception {
-        Organisation organisationSaved = organisationRepository.save(new Organisation().setName(ORG_NAME).setDescription(ORG_DESCRIPTION));
+        Organisation organisationSaved = organisationRepository.save(new Organisation()
+                .setName(ORG_NAME)
+                .setDescription(ORG_DESCRIPTION));
         catalogRepository.save(new Catalog().setName(CAT_NAME).setOrganisation(organisationSaved));
         BundleGroup stubBundleGroup = getStubBundleGroup(true, organisationSaved, Optional.empty());
 
         BundleGroupDto bundleGroupNoId = bundleGroupMapper.toDto(stubBundleGroup);
-
+        bundleGroupNoId.setBundleGroupId(null);
```

Comment:
Is this code required?

Reply:
Yes, because the intention of the developer was to provide an object of type BundleGroup with no ID. The stubbed objcect has a default ID (BUNDLE_GROUP_ID), so to respect the original will of the developer I forced it to null.